modules/openapi-generator/src/main/resources/typescript/model/ObjectSerializer.mustache

```diff
@@ -59,6 +49,45 @@ let typeMap: {[index: string]: any} = {
     {{/models}}
 }
 
+type MimeTypeDescriptor = {
```

Comment:
can you please add docs to these types/properties/functions?

Reply:
@macjohnny, sure! Just to clarify, am I correct in understanding that we're talking about comments in the code, not some other docs, right?

Reply:
Yes, code comments are fine

Reply:
Done